api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
I've been thinking through the idea of pushing the logic for extracting mentions from JavaScript down to Python. We'd still [convert]() `<span class="atwho">` to markdown before saving, but then we'd extract mentions from the markdown on the server side. The purpose would be to simplify the API: we wouldn't need `*_mentions` in the API at all.

That raises a bit of a question as to whether we would also push the conversion of `[@foo](/barry/)` into `<span class="atwho">` down into Python. I think asymmetry would make sense here, since `atwho` itself is totally a client-side concern.

On the other hand, we already have to parse out the mention guids on the client side as part of the conversion from `atwho` to markdown, so maybe it makes sense to keep doing the extraction there and pass `new_mentions` in over `POST`/`PUT`.

@leb2dg Waddya think? :-)